x-pack/test/security_solution_cypress/cypress/e2e/detection_response/rule_creation/override.cy.ts

```diff
@@ -146,8 +146,8 @@ describe('Rules override', { tags: ['@ess', '@serverless'] }, () => {
     cy.get(ALERTS_COUNT)
       .invoke('text')
       .should('match', /^[1-9].+$/); // Any number of alerts
-    cy.get(ALERT_GRID_CELL).contains('auditbeat');
-    cy.get(ALERT_GRID_CELL).contains('critical');
+    cy.get(ALERT_GRID_CELL).contains('winlogbeat');
```

Comment:
Hey, why this is changing?

Reply:
don't remember, let me look into this again real quick!

Reply:
@nkhristinin ok I know why this value had to change: the new `auditbeat_single` file now used in all tests (unless the test specifically load `auditbeat_multiple` has the following value:

```
"agent": {
  "type": "winlogbeat",
  ...
}
```

That's it :)